code/index.html

```diff
@@ -1,22 +1,103 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compaatible" content="i.e-edge">
+    <title>I did it - The News Site</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
+</head>
+
+
+<header>
+    <div class="logo">
+        <img src="logo.svg" class="technico">
+    </div>
+
+
+
+    <a class="nav item1" href="#">No turnig back now</a>
+    <a class="nav item1" href="#">What have I done</a>
+    <a class="nav item1" href="#">Going nuts</a>
+    <a class="nav item1" href="#">I quit</a>
+
+    <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+    </div>
+
+
+</header>
+
+
+<main class="papercontainer">
+    <section class="anything">
+        <article class="card topstroy">
+            <img src="https://cdn.pixabay.com/photo/2016/06/15/16/00/woman-1459220_1280.png" alt="Kvinna vid dator">
+            <div class="topstory--rightcol">
+                <h1>Learn to code in 24 weeks</h1>
+                <p id="topstory">☝🏼 Enter a Bootcamp they said.<br>🤙🏼 You don't have to have any pre-knowledge they said.<br><br> Well that is if you haven't tested to code online or maybe worked as a google analytic or maybe you just have a developer friend hanging
+                    around that can help you out.<br> I heard this talk about rollercoaster rides 🎢 but I never realized that you actually could get stuck in the loop and just hang there upside down, full of anxiety and fear!<br> Sunday isn't over yet
+                    👊🏻. Cross your fingers🤞🏼!
+                </p>
+                <div class="topstory--linkbox">
+                    <a href="https://www.technigo.io/program" class="redlink" , target="_blank">Read more</a>
+                </div>
+            </div>
+        </article>
     </section>
```

Comment:
Perfect indentation here, which makes it easier to follow what elements are nested etc 👍